src/lib/form/validators.ts

```diff
@@ -192,6 +193,22 @@ export class Validators {
     return Validators.pattern(errorMessage, emailRegex);
   }
 
+  public static phoneNumber(errorMessage: string): ValidationRule {
+    const applies: ValidatorFn = (value: string) => {
+      if (Validators.required("").applies(value)) return false;
+
+      try {
+        const phoneValidator = PhoneNumberUtil.getInstance();
+        return !phoneValidator.isValidNumber(phoneValidator.parse(value, "GB"));
+      } catch (error) {
+        // Phone number could not be parsed, i.e. is invalid
+        return true;
+      }
+    };
+
+    return { errorMessage, applies };
+  }
+
```

Comment:
@NyWeb I wonder what you think about this comment 😂   I'm happy to remove.

See Virtual Visit's [phone number validator](https://github.com/madetech/nhs-virtual-visit/blob/7bc85b4e45b67a991054b91547136a76d063e9b9/src/helpers/validateMobileNumber.js).